File: 1 занятие/c.py
with open("input.txt") as file:
    f, c = map(int, file.readline().split())

with open("output.txt", "w") as file:
    file.write(str(5 / 9 * (f - 32)))
    file.write(" ")
    file.write(str(9 / 5 * c + 32))

File: 1 занятие/c.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    ifstream in("input.txt");
    ofstream out("output.txt");

    long double f, c;
    in >> f >> c;
    out << fixed << setprecision(10);
    out << 5.0 / 9.0 * (f - 32.0);
    out << " " << 9.0 / 5.0 * c + 32.0;
}